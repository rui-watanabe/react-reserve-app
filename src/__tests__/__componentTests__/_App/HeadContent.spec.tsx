import React from 'react';
import renderer from 'react-test-renderer';
import HeadContent from '../../../components/_App/HeadContent';

test('HeadContent', () => {
  const component = renderer.create(<HeadContent />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
