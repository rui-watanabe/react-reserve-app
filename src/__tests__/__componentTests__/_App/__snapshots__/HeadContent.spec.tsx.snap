// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadContent 1`] = `
Array [
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />,
  <meta
    charSet="utf-8"
  />,
  <link
    href="/static/favicon/apple-touch-icon.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />,
  <link
    href="/static/favicon/favicon-32x32.png"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />,
  <link
    href="/static/favicon/favicon-16x16.png"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />,
  <link
    href="/static/favicon/site.webmanifest"
    rel="manifest"
  />,
  <link
    color="#0041df"
    href="/static/favicon/safari-pinned-tab.svg"
    rel="mask-icon"
  />,
  <meta
    content="#00df94"
    name="msapplication-TileColor"
  />,
  <meta
    content="#00df94"
    name="theme-color"
  />,
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />,
  <meta
    charSet="utf-8"
  />,
  <link
    href="/static/favicon/apple-touch-icon.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />,
  <link
    href="/static/favicon/favicon-32x32.png"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />,
  <link
    href="/static/favicon/favicon-16x16.png"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />,
  <link
    href="/static/favicon/site.webmanifest"
    rel="manifest"
  />,
  <link
    color="#0041df"
    href="/static/favicon/safari-pinned-tab.svg"
    rel="mask-icon"
  />,
  <meta
    content="#00df94"
    name="msapplication-TileColor"
  />,
  <meta
    content="#00df94"
    name="theme-color"
  />,
]
`;
